docs/cloud/glossary.md

```diff
@@ -0,0 +1,439 @@
+Accounts.
+
+:::tip Accounts vs Users
+
+To avoid confusion, "User" is the term for users in a Directus Project, while the terms "Account" and "Team Member" are
+used in the context of Directus Cloud. Accounts on Directus Cloud and Users in a Directus Project are two separate
+systems.
+
+:::
+
+## Asset Storage
+
+Directus Core allows asset storage for any type of file. This applies on all Cloud Projects. However, on Directus Cloud,
+certain hard technical limits apply to file uploads such as limits on size per asset and total asset storage per
+Project. For more details, see [Cloud Policies](https://directus.io/cloud-policies/#).
+
+Asset Storage space varies by Project Tier. For details, see [Cloud Policies](https://directus.io/cloud-policies/).
+
+## Automatic Software Updates
+
+Software updates on any app can be a time-consuming chore, taking developer energy and focus away from core business
+logic. Cloud Projects are updated every two weeks in rolling releases _(i.e. done with no downtime)_. These updates keep
+your Project on the latest version and fully up-to-date with all [Cloud Exclusives](#cloud-exclusives), dependencies,
+and security patches.
+
+::: tip Version-locking
+
+Version-locking is available on Enterprise Projects.
+
+:::
+
+## Backups
+
+By default, Standard and Enterprise Project data is backed up once per day, but the frequency can be configured as
+needed on Enterprise Projects. In the event of an emergency, your data will be safe.
+
+## Billing
+
+Community Projects are free. Each Enterprise Project is custom-tailored and so is the bill. For more information on
+Enterprise billing, [contact sales](https://directus.io/contact/). The rest of this section will focus on billing
+details for Standard Projects.
+
+### Manage Billing
+
+Billing is organized such that each Team has its own separate billing and each Project within a Team is
+[calculated](#how-bills-are-calculated) and [invoiced separately every month](#billing-cycles).
+
+Team Members are SuperAdmins. As such, they have full access to
+[manage billing information and payment methods](/cloud/teams/#manage-billing), including information provided by other
+Team Members. Team Members should be highly trusted individuals.
+
+All bills will first be invoiced to the default payment method. You may notice that it is possible to add additional
+payment methods, and it may be a good idea to do so, as keeping multiple payment methods on file provides a fail-safe to
+help make sure your Project stays running. In the event that there is an issue processing the default payment method,
+the other payment methods on file will be attempted. If the bill is not paid, the Nodes are paused, halting web-traffic,
+until a successful payment is made. If the paused Project is never repaid, it will eventually be deleted along with all
+its data and assets! For more details see, [Cloud Policies](https://directus.io/cloud-policies/).
```

Comment:
I think that information might be better suited for a terms and conditions, rather than the main glossary of the docs 🤔 We should write against key journey/golden path for the main docs imo. Thoughts @benhaynes?

Reply:
> I think that information might be better suited for a terms and conditions

I agree btw, this tidbit just definitely needs to be *somewhere*